README.md

```diff
@@ -935,6 +936,18 @@ expect(message).to eq({
   topic: 'my-topic',
   key: 'my-id'
 })
+
+## Configuring Deimos to test settings
```

Comment:
So this shouldn't be in the code block with everything else because it likely won't be in the test itself. Also, the previous examples in this block will *only* work if the test producers are being used, so we should call that out and put this explanation before that block. 

You can give an example of a `spec_helper` that does it:

```ruby
# spec_helper.rb
RSpec.before(:each) do
  configure_deimos
end
```

Actually, now that I write this, it would be even better if we can allow this method to take a block and reset the config back to what it was. Then you could use it like this:

```ruby
RSpec.around(:each) do |ex|
  configure_deimos({...}) do
    ex.run
  end
```

...and we would be able to ensure that config didn't leak past examples, which is something that's bitten me multiple times. You can deep-dup the config and restore it after the block is run.